jpa/platform-tests/src/main/java/ee/jakarta/tck/persistence/core/criteriaapi/CriteriaBuilder/Client1AppmanagedTest.java

```diff
@@ -125,6 +126,7 @@ public static EnterpriseArchive createDeploymentVehicle(@ArquillianResource Test
                 com.sun.ts.tests.common.vehicle.VehicleClient.class,
                 com.sun.ts.tests.common.vehicle.ejb3share.NoopTransactionWrapper.class
             );
+            jpa_core_criteriapia_CriteriaBuilder_appmanaged_vehicle_ejb.addClasses(ee.jakarta.tck.persistence.common.schema30.Util.getSchema30classes());
```

Comment:
We can instead call on the previous addClasses call so it would look like:
something jar.addClasses(ee.jakarta.tck.persistence.common.schema30.Util.class, ... )
.addClasses(ee.jakarta.tck.persistence.common.schema30.Util.getSchema30classes());